Documentation/cmdref/clustermesh-apiserver_kvstoremesh.md

```diff
@@ -16,10 +16,11 @@ clustermesh-apiserver kvstoremesh [flags]
       --cluster-name string                          Name of the cluster. It must consist of at most 32 lower case alphanumeric characters and '-', start and end with an alphanumeric character. (default "default")
       --clustermesh-config string                    Path to the ClusterMesh configuration directory
       --controller-group-metrics strings             List of controller group names for which to to enable metrics. Accepts 'all' and 'none'. The set of controller group names available is not guaranteed to be stable between Cilium versions.
+      --crd-wait-timeout duration                    Cilium will exit if CRDs are not available within this duration upon startup (default 5m0s)
   -D, --debug                                        Enable debugging mode
       --enable-gops                                  Enable gops server (default true)
       --global-ready-timeout duration                KVStoreMesh will be considered ready even if any remote clusters have failed to synchronize within this duration (default 10m0s)
-      --gops-port uint16                             Port for gops server to listen on (default 9894)
+      --gops-port uint16                             Port for gops server to listen on (default 9892)
```

Comment:
This change should be reverted, because the default port used by ClusterMesh needs to be different compared to the one used by KVStoreMesh. Otherwise, both processes would listen to the same port (which apparently doesn't lead to an explicit error because [ReuseSocketAddrAndPort is set](https://github.com/cilium/cilium/blob/pr%2Fjoe%2Fclustermesh-hive/pkg/gops/cell.go#L50-L53)), hence leading to unexpected behavior.

Reply:
Understood. I wonder if I can find a way to parameterize these, or we'll just retain the duplicate definitions.